### PR TITLE
Single user page

### PR DESCRIPTION
**Pull request by Ankur9669**
Created Single User Page
image

The css is already reviwed so you can omit that

### CODE REVIEW

File: fitbit-react/src/Util/Toasts/toasts.css [hunk 1]
```diff
@@ -11,6 +11,10 @@
 .toast-text{
     flex: 1;
 }
+.toast-info{
+    background-color: var(--blue);
+    color: white;
```
Comment (pradipgharge) [inline fitbit-react/src/Util/Toasts/toasts.css]: you can use color variable here

**Review by pradipgharge — APPROVED** [review]
The Ui looks really nice and the react code is also clean.
Looks good to me for merging

**Review by sri-rishi — APPROVED** [review]
UI and code looks good, just one suggestion when user logged out he/she must redirected to login or home page, as per I think user profile page should not be accessible without login

File: fitbit-react/src/Util/Toasts/toasts.css (hunk 1 above)
Comment (Ankur9669) [reply fitbit-react/src/Util/Toasts/toasts.css]: Done

**Pull request merged**